readme.md

```diff
@@ -47,7 +47,7 @@ Just some easy ones.
 
 ### As a Guitar Player, I want to:
 1. Tab a famous guitar lick so I can send it to my friend to help him learn the lick.
-2. Share my lick publically so everyone can hear it
+2. Share my lick publicly so everyone can hear it
```

Comment:
Well, some standard dictionaries do include “publically” as a variant spelling, but it’s described as less popular than “publicly.” In fact, “publicly” outnumbers “publically” by more than 100 to 1 in Google searches. see:  https://www.grammarphobia.com/blog/2015/08/publicly-publically.html